README.md

```diff
@@ -45,9 +46,25 @@ class Home(DjangoVueComponent):
     vue_data = {'count': 0}
 ```
 
-## Roadmap
+```HTML
+{% /templates/components/counter.html %}
```

Comment:
Is this supposed to be a comment or a tag? I think you meant `{% comment ... %}` or something like `<!-- ... -->`? Very nice additions so far!